integration-tests/cli/build.gradle.kts

```diff
@@ -12,6 +12,8 @@ evaluationDependsOn(":plugins:base")
 dependencies {
     implementation(kotlin("stdlib"))
     implementation(kotlin("test-junit"))
+    val jackson_version: String by project
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jackson_version")
```

Comment:
after json parsing was moved back to core, is this needed anymore?

Reply:
My fault, you're right